### DISCUSSION TITLE
Range measurement between two crazyflies

### DISCUSSION THREAD

**Discussion by isro01** [bitcraze/discussions#147] [Q&A]
Hi
So I'm working with a system of two crazyflies each with loco deck, and I would like to get the range distance between them (by supposedly making the decks act like anchors as well). I did find this discussion - https://github.com/bitcraze/crazyflie-firmware/issues/576, but I'm not sure I understand it entirely. Has this feature been implemented, and if not, is it possible to get the range between two crazyflies just using the loco deck?
Thanks

**Comment by knmcguire** [comment]
Hi! 

We have not such a functionality yet, but we hope to intergrate it in the future under then name Hybrid TDOA mode, which allows inter drone ranging (see this [blogpost](https://www.bitcraze.io/blog/)).

You can take a look at this closed PR: https://github.com/bitcraze/crazyflie-firmware/pull/580, which removes the full loco deck driver and replaces it with inter drone ranging.

Also I would suggest to wait a week for @krichardsson to come back from his holiday in a week, so I'll ping him once he is here.

**Comment by krichardsson** [comment]
Hi!

> is it possible to get the range between two crazyflies just using the loco deck

Yes, in theory it is possible but it is unfortunately not implemented yet. I wrote an example implementation (referenced as a gist from the github issue) but it is not complete and I think there was an error in it as well. I am hoping to maybe have time to do it in the coming weeks, but I don't want to promise anything. For now, the gist is the best source I think.
